lib/org/chromium/webidl/FetchSpecRunner.js

```diff
@@ -19,18 +20,26 @@ foam.CLASS({
 
   properties: [
     {
+      class: 'String',
+      documentation: 'The n:m relationship type of input-to-output.',
+      name: 'ioRelationshipType',
+      value: '1:(0|1)',
```

Comment:
If we ever parse these out, we are unlikely to support this value. Probably better to say `'1:1'` and add a comment mentioning that it's actually one-to-one-or-zero (and explain why).